src/main/java/com/sucy/skill/dynamic/trigger/ConsumeTrigger.java

```diff
@@ -30,6 +30,11 @@ public String getKey() {
     public boolean shouldTrigger(PlayerItemConsumeEvent event, int level, Settings settings) {
         final String item_name = event.getItem().getType().name();
         final String req = settings.getString("material", "Any").replace(" ","_");
+        if (item_name.equalsIgnoreCase("potion") && req.equalsIgnoreCase(item_name)){
```

Comment:
This also doesn't account for splash/lingering potions. Perhaps using `item_name.contains` would work instead.

Reply:
Oh. Haha I guess you're right. Lol. My bad 🤣